lms/djangoapps/shoppingcart/api.py

```diff
@@ -23,9 +23,6 @@ def order_history(user, **kwargs):
     for order_item in purchased_order_items:
         # Avoid repeated entries for the same order id.
         if order_item.order.id not in [item['order_id'] for item in order_history_list]:
-            # If we are in a Microsite, then include the orders having courses attributed (by ORG) to that Microsite.
-            # Conversely, if we are not in a Microsite, then include the orders having courses
-            # not attributed (by ORG) to any Microsite.
```

Comment:
This comment is just out of date? And the the `course_org_filter` and `org_filter_out_set` kwargs are more generic for normal site configs now? Seems that way, just confirming.

Also, the way those two kwargs are defined is very unusual - manually checking kwargs rather than having them in the function prototype. Also they should use `.get()` with a default rather than a tertiary conditional? And the giant multi-stage conditional below this code could use some temporary variables or something. I don't like this function. But that's not super relevant to the task at hand.

Reply:
Right, this comment seemed out of date to me as well.
Besides, this entire django app should be removed in favor of the ecommerce service, as written in this PR: https://github.com/edx/edx-platform/pull/20711